### ISSUE TITLE
Support for double angle bracket >> 

### ISSUE THREAD

**Issue by TimCreasman** [sindresorhus/execa#1164]
Hello!

Apologies if I missed this in the documentation or the closed issues, but is there support for the `>>` redirection operator? 

From my understanding, the single `>` bracket is essentially supported via the `execa({stdout: {file: 'output.txt'}})` syntax but I couldn't find anything referencing `>>`.

`>>` is supposed to append the stdout to a file so maybe a syntax like `execa({stdout: {append: 'output.txt'}})` would make sense?

**Comment by ehmicky** [comment]
Hi @TimCreasman,

Execa [does not use shells](https://github.com/sindresorhus/execa/blob/main/docs/shell.md), so it does not use shell-specific syntax like `>>`.

That being said, we do try to emulate shell features. In this case, this would be appending to a file.

You can achieve this by doing:

```js
await execa(..., {stdout: [createWriteStream('output.txt', {flags: 'a'}), 'pipe']})
```

Now, that's a little verbose, so we could maybe add the following syntax instead:

```js
await execa(..., {stdout: {file: 'output.txt', append: true}})
```

What do you think @sindresorhus?

**Comment by TimCreasman** [comment]
@ehmicky 

Full disclosure, I did not understand that so thanks for the clarification. I see now this project essentially wraps node's `child_process` functionality (and if I had read the very first paragraph of this repo I would already have known that haha). I only just started using execa this week having moved from shelljs - loving it so far 👍 

And thanks for the example! The second syntax example would be very nice but its good knowing the functionality is already easily available and not crazy verbose.

**Comment by sindresorhus** [comment]
> What do you think @sindresorhus?

👍

**Comment by ehmicky** [comment]
Done at #1166.

**Comment by ehmicky** [comment]
Published in [`9.5.0`](https://github.com/sindresorhus/execa/releases/tag/v9.5.0).
